darkhttpd.c

```diff
@@ -966,6 +966,9 @@ static void usage(const char *argv0) {
 #endif
     printf("\t--no-keepalive\n"
     "\t\tDisables HTTP Keep-Alive functionality.\n\n");
+    printf("\t--single-file\n"
+    "\t\tOnly serve a single file provided as /path/to/wwwroot instead\n"
```

Comment:
Maybe `/path/to/wwwfile` would be more clear?

Reply:
Thought about it too. The problem is that every description so far assumes a directory. I don't think changing that for this niche option is a good thing. And mentioning wwwfile only once could be confusing too. Furthermore wwwroot could stand for www-root-file and not www-root-directory, so it's not wrong.

I am open to changing it if you want

Reply:
Understandable! You could definitely leave it as-is.

Reply:
How about `/path/to/file`

Reply:
Changed it